packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/impactanalysis/services/ImpactAnalysisMessageService.java

```diff
@@ -1,70 +0,0 @@
-/*******************************************************************************
```

Comment:
We don't need this class anymore.
All the information it provided in the messages is now visible in the tree.